management/src/main/java/org/kaazing/gateway/management/monitoring/configuration/impl/DefaultMMFMonitoringDataManagerStub.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2007-2014 Kaazing Corporation. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.kaazing.gateway.management.monitoring.configuration.impl;
+
+import org.kaazing.gateway.management.monitoring.configuration.MonitoringDataManager;
+import org.kaazing.gateway.management.monitoring.entity.manager.impl.ServiceCounterManagerImpl;
+import org.kaazing.gateway.management.monitoring.service.MonitoredService;
+import org.kaazing.gateway.service.DefaultLongMonitoringCounterStub;
+import org.kaazing.gateway.service.LongMonitoringCounter;
+import org.kaazing.gateway.service.MonitoringEntityFactory;
+
+public class DefaultMMFMonitoringDataManagerStub implements MonitoringDataManager {
```

Comment:
This class is just a no-op implementation of MonitoringDataManager. I would therefore drop MMF from the name since it has nothing to do with MMF. Also, going forward it won't be the default (we will want gathering statistics in MMF to be the default, see metrics configuration story), so I would recommend also removing the word Default from the name. That leaves us with the name MonitoringDataManagerStub. You could also perhaps consider the name NoopMonitoringDataManager or DoNothingMonitoringDataManager.